examples/nextjs-live-avatars-advanced/pages/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { RoomProvider } from "../liveblocks.config";
+import { useRouter } from "next/router";
+import LiveAvatars from "../components/LiveAvatars";
+import { useMemo } from "react";
+import styles from "../styles/Index.module.css";
+
+export default function Example() {
+  const roomId = useOverrideRoomId("nextjs-live-avatars-advanced");
+
+  return (
+    <RoomProvider id={roomId}>
+      <main className={styles.main}>
+        <LiveAvatars />
+      </main>
+    </RoomProvider>
+  );
+}
+
+export async function getStaticProps() {
+  const API_KEY = process.env.LIVEBLOCKS_SECRET_KEY;
+  const API_KEY_WARNING = process.env.CODESANDBOX_SSE
+    ? `Add your public key from https://liveblocks.io/dashboard/apikeys as the \`LIVEBLOCKS_SECRET_KEY\` secret in CodeSandbox.\n` +
+    `Learn more: https://github.com/liveblocks/liveblocks/tree/main/examples/nextjs-live-cursors#codesandbox.`
+    : `Create an \`.env.local\` file and add your public key from https://liveblocks.io/dashboard/apikeys as the \`LIVEBLOCKS_SECRET_KEY\` environment variable.\n` +
+    `Learn more: https://github.com/liveblocks/liveblocks/tree/main/examples/nextjs-live-cursors#getting-started.`;
+
+  if (!API_KEY) {
+    console.warn(API_KEY_WARNING);
+  }
+
+  return { props: {} };
+}
```

Comment:
Since this example is using a private key and there's already a warning in `auth.ts` we can remove this.